addons/auth_signup/views/auth_signup_login_templates.xml

```diff
@@ -61,11 +61,11 @@
 
         <template id="auth_signup.reset_password" name="Reset password">
             <t t-call="web.login_layout">
-                <div t-if="message">
+                <div t-if="message" class="oe_login_form">
                     <p class="alert alert-success" t-if="message" role="status">
                         <t t-esc="message"/>
                     </p>
-                    <a href="/web/login" class="btn btn-link float-right" role="button">Back to Login</a>
+                    <a href="/web/login" class="btn btn-link justify-content-end mt-2 d-flex small" role="button">Back to Login</a>
```

Comment:
`btn small` should not be used. You should add the `btn-sm` class instead which is there for that purpose.
To align the button to the right, I would either wrap the button in a div with text-right alignement or keep the float-right and add clearfix on the oe_login_form element.